### PR TITLE
Authorization management refresh for Credentials

### PR DESCRIPTION
**Pull request by atstojanov**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/cherrytwist/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/cherrytwist/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/cherrytwist/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/components/Community/UserCard.tsx [hunk 1]
```diff
@@ -39,12 +39,14 @@ const UserCardInner: FC<UserCardProps> = ({ name, terms, id }) => {
...
 
   const data = userData?.usersById[0] as User;
-  const groups = data?.memberof?.communities
-    ?.flatMap(c => {
-      if (c) return c.groups?.map(g => g.name.toLowerCase());
-      return undefined;
-    })
-    .filter((x): x is string => x !== undefined);
+  const groups = [] as string[];
+  // TODO [ATS]: Resolve this
```
Comment (valentinyanakiev) [inline src/components/Community/UserCard.tsx:43]: Is that resolved?

File: src/components/UserProfile/UserProfile.tsx [hunk 2]
```diff
@@ -165,19 +163,19 @@ export const UserProfile: FC = () => {
...
 
   const references = user?.profile?.references || [];
 
-  const groups =
-    user?.memberof?.communities
-      .flatMap(
-        c => c && c.groups && c?.groups.map(x => (c.type !== CommunityType.ECOVERSE ? `${x.name} (${c.name})` : x.name))
-      )
-      .filter((x): x is string => x !== undefined) || [];
+  const groups = [] as string[]; // TODO [ATS]: Finish after resourceId unification
```
Comment (valentinyanakiev) [inline src/components/UserProfile/UserProfile.tsx:170]: Is this still staying as a comment?

**Review by valentinyanakiev — COMMENTED** [review]
There are a couple of comments that are left behind but overall the code looks clean - if the comments are intentionally left out the code can go in.

File: src/components/UserProfile/UserProfile.tsx (hunk 2 above)
Comment (atstojanov) [reply src/components/UserProfile/UserProfile.tsx:170]: Yes

File: src/components/Community/UserCard.tsx (hunk 1 above)
Comment (atstojanov) [reply src/components/Community/UserCard.tsx:43]: No. There should be an issue about the group management changes as we discussed yesterday.

**Review by valentinyanakiev — APPROVED** [review]

**Pull request merged**